### DISCUSSION TITLE
Lightning Flash 0.3!

### DISCUSSION THREAD

**Discussion by edenlightning** [Lightning-AI/pytorch-lightning#7719] [General]
Lightning Flash is a toolkit designed to make baselines and common deep learning tasks as simple as possible with minimum coding and maximum capabilities. We’re proud to share a new version of flash, with 8 new tasks across NLP and vision:
Image Embedder
Multi-label Image Classification
Summarization
Translation
Object Detection
Video Classification
Semantic Segmentation
Style Transfer
Our new release also includes a modular API for data processing and registering models, so you can customize and tailor your tasks to any use case.

Blogpost: https://bit.ly/3vld9HA
GitHub: https://bit.ly/2RiAV7Z

![1_51DGsstZNd8pQnoO76uPWg](https://user-images.githubusercontent.com/66261195/119638576-791adf80-be1f-11eb-8487-6daec198bcdd.png)

**Comment by akihironitta** [comment]
We should unpin this discussion from https://github.com/PyTorchLightning/pytorch-lightning/discussions, since the latest version of Flash is currently `0.5.0`.
